targets/_example/profile/module.c

```diff
@@ -19,7 +19,7 @@ const servo_params_t servo_params = {
 void app_init_services() {
     // see jacdac-c/services/jd_services.h for the services that can be
     // enabled here you can enable zero or more services
-    servo_init(&servo_params);
+    servo_init(&servo_params, 0xff);
```

Comment:
I usually just pass -1

Reply:
Would you like me to change the parameter to expect -1?

Reply:
well, if it takes uint8_t then -1 and 0xff are the same

Reply:
maybe we should defined PIN_NC or something to be -1

Reply:
or maybe you're right, #define NO_PIN 0xff will be probably less error prone

Reply:
okie dokie!